### PR TITLE
Updated ThemePark

### PR DESCRIPTION
**Pull request by stijnb1234**
### Type
- [ ] Internal change (Doesn't affect end-user).
- [ ] External change (Does affect end-user).
- [x] Wiki (Changes towards the [Wiki]).
- [ ] Other: __________ 

### Description
Updated the ThemePark placeholders to v1.2 on eCloud.

[Wiki]: https://github.com/PlaceholderAPI/PlaceholderAPI/wiki

### CODE REVIEW

**Comment by stijnb1234** [comment]
My expansion is still not approved, so I added the url to the expansion.

File: wiki/Placeholders.md [hunk 1]
```diff
@@ -3965,11 +3965,13 @@ Per Player/Event placeholders:
 ----
 
 - ### **[ThemePark](https://www.spigotmc.org/resources/48648/)**
-> NO DOWNLOAD COMMAND
+> [Download here](https://api.extendedclip.com/expansions/themepark/)
```
Comment (Andre601) [inline wiki/Placeholders.md:3968]: Doesn't matter if it is yet approved or not. Just provide the command itself. People will be able to download it through multiple ways.
```suggestion
> /papi ecloud download themepark
```

**Review by Andre601 — CHANGES_REQUESTED** [review]

**Review by Andre601 — APPROVED** [review]

**Comment by stijnb1234** [comment]
Changed. Any idea when my expansion will be approved? Its already waiting more then 5 months.

**Comment by Andre601** [comment]
There isn't an ETA on when those will be approved or denied.

@Funny-cube, @extendedclip and I think @darbyjack manage the eCloud so one of them has to approve/deny it.

**Comment by Funny-cube** [comment]
Expansions are checked for verification upon request.

> If you've uploaded an expansion and wish for it to be verified contact us via our discord server.

In terms of the ThemePark expansions seems the author name and version in the expansion jar don't match what's listed on the eCloud, these should be matching to avoid any issues with update notifications and search.

**Comment by stijnb1234** [comment]
@Funny-cube That's fixed now. ;)

**Pull request merged**